bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/util/ByteStringer.java

```diff
@@ -54,6 +54,15 @@ public static ByteString wrap(final byte[] array) {
     return USE_ZEROCOPYBYTESTRING? ZeroCopyByteStringUtil.wrap(array): ByteString.copyFrom(array);
   }
 
+  /**
+   * Wraps a byte array in a {@link ByteString} without copying it.
+   */
+  public static ByteString wrap(final byte[] array, int offset, int length) {
+    return USE_ZEROCOPYBYTESTRING?
```

Comment:
I think I'm missing something here...
When adapting hbase's Result, isn't it always the case that offset !=0? So this should prevent 3 array copies per cell?